lua/hop/init.lua

```diff
@@ -93,6 +92,7 @@ local function hint_with(hint_mode, opts)
     opts
   )
 
+  local h
```

Comment:
Can you explicitly set it to `nil`, pretty please? :)

Reply:
Sure! (Why though? [Lua does automatic initialization to `nil`](https://www.lua.org/manual/5.1/manual.html#2.3))

Reply:
I guess it’s fine then, but it was just a matter of coherency with the rest of the codebase.